packages/alfa-rules/src/sia-r15/rule.ts

```diff
@@ -61,11 +63,23 @@ export default Rule.Atomic.of<Page, Iterable<Element>, Question>({
             }, Map.empty<Option<string>, List<Element>>())
             .values()
         );
+        return filter(groups, (group) => group.size > 1);
       },
 
       expectations(target) {
+        // @see https://html.spec.whatwg.org/multipage/iframe-embed-object.html#process-the-iframe-attributes
         const sources = Set.from(
-          map(target, (iframe) => iframe.attribute("src"))
```

Comment:
Looking only at the attribute was bad as two attributes with the same value end up being different, thus we were never hitting the "same resource" case.